src/elements/overlay/overlay.visual.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { html, type TemplateResult } from 'lit';
```

Comment:
Would it be possible to have a scenario where the scrollbars are visible?

Reply:
Done